test/js/animation.js

```diff
@@ -318,12 +318,12 @@ suite('animation', function() {
     tick(1000);
     var a = document.body.animate([], 2000);
     a.finish();
-    assert.equal(a.startTime, 0);
+    assert.equal(a.startTime, -1000);
```

Comment:
If I understand correctly, animate() should cause the animation to start playing, in which case startTime would be 1000, and endTime would be calculated from that instead. Is this a difference between animation time and document timeline time?